src/app/services/sms/sms.types.ts

```diff
@@ -60,9 +60,12 @@ export interface IVerificationSmsCount extends ISmsCount {
     email: string
     userId: IUserSchema['_id']
   }
+  isOnboardedAccount: boolean
 }
 
-export type IVerificationSmsCountSchema = ISmsCountSchema
+export type IVerificationSmsCountSchema = ISmsCountSchema & {
```

Comment:
this seems to be a leftover bug. it should extend from `IVerificationSmsCount` instead.